Makefile

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
```

Comment:
I'm the opposite and use Makefiles almost anywhere :-) What is your preferred way of running such commands? Just documentation or some shell scripts? I can change it to your preferred way.